commands/logs_e2e_test.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mmctl/client"
+
+	"github.com/spf13/cobra"
+)
+
+func (s *MmctlE2ETestSuite) TestlogsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Display single log line", func(c client.Client) {
```

Comment:
We should probably want this case to run with only system admin privileges.

```suggestion
	s.RunForSystemAdminAndLocal("Display single log line", func(c client.Client) {
```